### PR TITLE
Added cask for tales of majeyal

### PR DESCRIPTION
**Pull request by dargonforce**

### CODE REVIEW

File: Casks/tales-of-majeyal.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'tales-of-majeyal' do
+  version '1.3.1'
+  sha256 '4f63480762ac783f60a93cb4e9a6b07cde772f62f1730f05915872569660469a'
+
+  url 'http://te4.org/dl/t-engine/t-engine4-osx-1.3.1.zip'
+  name 'Tales of Maj\'Eyal'
+  homepage 'http://te4.org/'
+  license :gpl
+
+  # Use of target to prevent conflicts with other T-Engine games
```
Comment (vitorgalvao) [inline Casks/tales-of-majeyal.rb]: Wrong comment. Please follow [the examples](http://i.imgur.com/A3DjUnv.png).

File: Casks/tales-of-majeyal.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'tales-of-majeyal' do
+  version '1.3.1'
+  sha256 '4f63480762ac783f60a93cb4e9a6b07cde772f62f1730f05915872569660469a'
+
+  url 'http://te4.org/dl/t-engine/t-engine4-osx-1.3.1.zip'
```
Comment (fanquake) [inline Casks/tales-of-majeyal.rb]: Can you use #{version} here?

Comment (ruseman) [inline Casks/tales-of-majeyal.rb]: @fanquake quite possibly.  I'm not sure how to do that however.

Comment (vitorgalvao) [inline Casks/tales-of-majeyal.rb]: @fanquake This is why we’re not supposed to actually ask for such a change. We either do it and inform the contributor, or merge as is and make the change. If the cask is correct and functional, it should not be held up.

I’ve merged it https://github.com/caskroom/homebrew-cask/commit/da5825d24a17956ed24bb7437d1383b2cf2f0a07, @ruseman, with the fix and squashed commits. It’s still credited to you. Thank you.

**Comment by ruseman** [comment]
I feel like I'm being quite slow about these.  I'm trying to move the majority of the applications I use to cask, so I hope these are helpful.

**Pull request closed without merging**